docs/release_notes/v0.16.1.rst

```diff
@@ -0,0 +1,11 @@
+v0.16.1
+=======
+
```

Comment:
A good practice is to have a simple narative summary sentence or paragraph to head one of these.